routes/firebase.js

```diff
@@ -0,0 +1,9 @@
+const admin = require('firebase-admin');
+
+require('dotenv').config();
+
+const credentials = require('../firebase-adminsdk.json');
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '../firebase-adminsdk.json'.